### PR TITLE
A few minor updates

### PR DESCRIPTION
**Pull request by tas50**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -5,10 +5,10 @@
   action :purge
 end
 
-execute "Setup ChefDK as default ruby" do
+execute 'Setup ChefDK as default ruby' do
   command "echo 'eval \"$(chef shell-init bash)\"' >> /root/.bashrc"
-  user "root"
-  environment( { "HOME" => "/root" } )
+  user 'root'
+  environment('HOME' => '/root')
```
Comment (jeremiahsnapp) [inline recipes/default.rb:11]: I'm not sure that it was the right thing to remove the braces here so I'll add them back in a separate commit.

File: recipes/dev-lxc.rb [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
 package 'lxc-dev'
 
-execute "install dev-lxc gem" do
-  command "chef gem install dev-lxc"
-  user "root"
-  environment( { "HOME" => "/root" } )
+execute 'install dev-lxc gem' do
+  command 'chef gem install dev-lxc'
+  user 'root'
+  environment('HOME' => '/root')
```
Comment (jeremiahsnapp) [inline recipes/dev-lxc.rb:8]: I'm not sure that it was the right thing to remove the braces here so I'll add them back in a separate commit.

**Comment by jeremiahsnapp** [comment]
Thanks a lot @tas50 for the updates. Much appreciated!

File: recipes/dev-lxc.rb (hunk 2 above)
Comment (tas50) [reply recipes/dev-lxc.rb:8]: It's a hash even without the {}s so this should be fine

**Pull request merged**